### PR TITLE
[filebeat] additionals labels

### PR DESCRIPTION
**Pull request by dugouchet**
- [x] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x] README.md updated with any new values or changes
- [x] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

File: filebeat/README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ helm install --name filebeat elastic/filebeat --version 7.2.0 --set imageTag=7.2
...
 | `podAnnotations`         | Configurable [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) applied to all Filebeat pods                                                                                                                                     | `{}`                                                                                                                      |
+| `labels`                 | Configurable [label](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) applied to all Filebeat pods                                                                                                                                     | `{}`                                                                                                                      |
```
Comment (Crazybus) [inline filebeat/README.md]: I think it should be "Configurable labels" not "label"

**Comment by Crazybus** [comment]
jenkins test this please

File: filebeat/README.md (hunk 1 above)
Comment (dugouchet) [reply filebeat/README.md]: thanks @Crazybus, I kept the same name as most of charts. Do you want me to change ?

Comment (Crazybus) [reply filebeat/README.md]: So I meant writing "Configurable labelS applied to all FIlebeat pods" instead of just "label".

**Comment by Crazybus** [comment]
Thank you for adding this in so quickly and for also adding in tests! ⭐️

File: filebeat/README.md (hunk 1 above)
Comment (dugouchet) [reply filebeat/README.md]: ho sorry

**Pull request merged**